src/publish.ts

```diff
@@ -55,12 +56,18 @@ const publish = async ({ extensionId, target, asset }: PluginConfig) => {
   if (!publishRes.status.includes('OK')) {
     const errors: SemanticReleaseError[] = []
     for (let i = 0; i < publishRes.status.length; i += 1) {
-      const message = publishRes.statusList[i]
       const code = publishRes.status[i]
-      const err = new SemanticReleaseError(message, code)
-      errors.push(err)
+      const message = publishRes.statusDetail[i]
+      if (code.includes('WARNING')) {
```

Comment:
According to the [publish API](https://developer.chrome.com/webstore/webstore_api/items/publish#response), `code` can only be one of the following:

* NOT_AUTHORIZED
* INVALID_DEVELOPER
* DEVELOPER_NO_OWNERSHIP
* DEVELOPER_SUSPENDED
* ITEM_NOT_FOUND
* ITEM_PENDING_REVIEW
* ITEM_TAKEN_DOWN
* PUBLISHER_SUSPENDED

Are you sure that `code` can also be `PUBLISHED_WITH_FRICTION_WARNING`?

Also, I think it would be better to have some sort of error whitelist array to check, instead of checking directly against the `code` variable, something like 

```js
const errorWhitelist = ['PUBLISHED_WITH_FRICTION_WARNING']

const publish = async [...] {
	[...]
	if (errorWhitelist.includes(code)) {
}
```